components/Cards/index.js

```diff
@@ -18,3 +18,74 @@
 // </div>
 //
 // Use your function to create a card for each of the articles and add the card to the DOM.
+let articlesArray = [];
+let cardsContainer = document.querySelector('.cards-container');
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+    .then((res) => {
+        console.log ("Res: ", res) 
+        articlesArray = res.data.articles;    
+        let javascriptArt = articlesArray.javascript;
+        let bootstraptArt = articlesArray.bootstrap;
+        let technologyArt = articlesArray.technology;
+        let jqueryArt = articlesArray.jquery;
+        let nodeArt = articlesArray.node;
+         
+        javascriptArt.forEach((data) => {
+            cardsContainer.appendChild(Cards(data));
+        });
+
+        bootstraptArt.forEach((data) => {
+            cardsContainer.appendChild(Cards(data));
+        });
+
+        technologyArt.forEach((data) => {
+            cardsContainer.appendChild(Cards(data));
+        });
+
+        jqueryArt.forEach((data) => {
+            cardsContainer.appendChild(Cards(data));
+        });
+
+        nodeArt.forEach((data) => {
+            cardsContainer.appendChild(Cards(data));
+        });
+
+       
+        console.log(javascriptArt);
+        // articlesArray.javascript.forEach((url) => {
+        // cardsContainer.appendChild(Cards(url));
+        // });
+    })
+    .catch( err => {
+        console.log("Error: ", err)
+    })
+    
+
+
+    function Cards(article) {
```

Comment:
Since this function is being defined in the global scope, it should be aligned with the `axios.get(` request and variable definitions above. Otherwise it looks, at a glance, like this function is being defined inside the .catch() or .then() functions (which we wouldn't want).

Summary: this shouldn't be indented.